app/Console/Commands/RussianRoulette.php

```diff
@@ -0,0 +1,376 @@
+        11¶¶1111¶¶11111111111111111111111111111111111
+        111¶11¶¶¶¶¶1111111111111111111111111111111111
+        11111¶¶11¶¶¶111111111111111111111111111111111
+        1111¶¶¶1111¶¶¶1111111111111111111111111111111
+        111¶¶¶¶¶¶1111¶¶¶11111111111111111111111111111
+        1111¶¶¶¶¶¶¶1111¶¶¶111111111111111111111111111
+        111111¶¶¶¶¶¶¶¶1111¶¶¶111111111111111111111111
+        11111111¶¶¶¶¶¶¶¶1111¶¶¶1111111111111111111111
+        11111111111¶¶¶¶¶¶¶¶111¶¶¶¶1111111111111111111
+        1111111111111¶¶¶¶¶¶¶¶111¶¶¶¶11111111111111111
+        111111111111111¶¶¶¶¶¶¶¶11¶¶¶¶¶¶11111111111111
+        111111111111111111¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶111111111111
+        1111111111111111111¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶111111111
+        1111111111111111111¶¶1¶¶¶1¶¶¶¶¶¶1¶¶¶¶¶¶111111
+        1111111111111111111¶1¶¶¶¶¶¶¶1¶¶¶1¶¶1¶¶¶¶11111
+        1111111111111111111¶¶¶¶¶¶¶¶¶¶¶11¶¶1¶¶1¶¶¶1111
+        1111111111111111111¶¶¶¶¶¶¶¶¶¶¶1¶¶¶¶¶¶¶11¶¶111
+        1111111111111111111¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶¶111
+        1111111111111111111111¶¶¶¶¶¶¶¶¶¶¶¶¶¶11¶11¶¶11
+        111111111111111111111111¶¶¶¶¶¶¶¶¶¶¶¶¶¶1¶¶¶¶¶¶
+        111111111111111111111111¶¶¶¶¶¶¶¶¶111¶¶¶¶111¶¶
+        111111111111111111111111¶¶¶111¶¶¶11111¶¶¶1111
+        111111111111111111111111¶¶¶111¶¶¶¶111111¶¶111
+        111111111111111111111111¶¶¶11¶¶¶¶¶11¶¶11¶1111
+        111111111111111111111111¶¶¶¶11¶11¶1¶¶¶¶1¶¶111
+        11111111111111111111111111¶¶¶¶11¶¶1¶¶¶¶¶1¶111
+        1111111111111111111111111111¶¶¶¶¶11¶¶¶¶¶1¶¶11
+        111111111111111111111111111111¶¶¶1¶¶¶¶¶¶1¶¶11
+        111111111111111111111111111111¶¶11¶¶¶¶¶¶1¶¶11
+        111111111111111111111111111111¶¶1¶¶¶¶¶¶¶1¶111
+        11111111111111111111111111111¶¶1¶¶¶¶¶¶¶¶1¶111
+        11111111111111111111111111111¶11¶¶¶¶¶¶¶1¶¶111
+        1111111111111111111111111111¶¶11¶¶¶¶¶¶11¶¶111
+        1111111111111111111111111111¶¶¶1111¶¶¶1¶¶1111
+        111111111111111111111111111111¶¶¶¶1111¶¶¶1111
+        11111111111111111111111111111111¶¶¶¶¶¶¶¶11111
+        ");
+        sleep(3);
+        system('clear');
+        $this->info("\n\t\t 
+         __        __   __                  __   __             ___ ___ ___  ___ 
+        |__) |  | /__` /__` |  /\  |\ |    |__) /  \ |  | |    |__   |   |  |__  
+        |  \ \__/ .__/ .__/ | /~~\ | \|    |  \ \__/ \__/ |___ |___  |   |  |___ 
+                                                                                 
+                           __                    ___                             
+                          / _`  /\   |\/|  |\/| |__                              
+                          \__> /~~\  |  |  |  | |___                             
+                                                                                 ");
+        $ans = $this->ask("\t\t\tAre you sure you want to play?");
+        while (!(('yes' == strtolower($ans) || 'y' == strtolower($ans)) || ('no' == strtolower($ans) || 'n' == strtolower($ans)))) {
+            $ans = $this->ask("\t\t
```

Comment:
Please do not use abbreviation it is very bad practice.

Reply:
i will try not to do this way in the future.